### PR TITLE
missing "-"

### PR DESCRIPTION
**Pull request by richardwhateverr**
I think this tag is ugly with no separation between solutions and the reasons but I fixed this typo.

### CODE REVIEW

**Review by daspositioned — DISMISSED** [review]

**Comment by rainbowdashlabs** [comment]
Maybe we could switch to embedded tags to make the tag overall look nicer c:

File: minecraft/old-combat-system.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ Du solltest keinen Minecraft 1.8(.8) Server verwenden aus mehreren Gründen.
 
 - <https://www.spigotmc.org/resources/protocolsupportstuff.44527/>
 
- <https://www.spigotmc.org/resources/oldcombatmechanics-disable-1-9-hit-cooldown.19510/>
+- <https://www.spigotmc.org/resources/oldcombatmechanics-disable-1-9-hit-cooldown.19510/>
```
Comment (Taucher2003) [inline minecraft/old-combat-system.md]: The formatting of the plugin links feels odd. The point in line 9 mentions that some plugins can restore the old combat system, but the plugins are at the same hierarchy than the message.

I would suggest to indent all 3 plugins links with 2 spaces to make them a subpoint of the last one.

**Review by Taucher2003 — DISMISSED** [review]
One comment, but I'm fine with merging this as an iterative improvement if we don't want to do the change in my comment.

**Comment by rainbowdashlabs** [comment]
i took the liberty to make the change myself and also removed a plugin which didn't receive update since over two years.

**Review by rainbowdashlabs — APPROVED** [review]

**Review by daspositioned — APPROVED** [review]

**Pull request merged**